src/app/modules/home/main/main.component.css.map

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What does this file do? I guess it maps the .css to .scss? Why not use .scss?

Reply:
I think it came from a SCSS plugin in my IDE. I removed it